exporter/signalfxexporter/eventclient.go

```diff
@@ -94,7 +94,10 @@ func (s *sfxEventClient) pushLogsData(ctx context.Context, ld plog.Logs) (int, e
 	}
 
 	defer func() {
-		io.Copy(ioutil.Discard, resp.Body)
+		_, err := io.Copy(ioutil.Discard, resp.Body)
+		if err != nil {
+			log.Fatalln(err)
```

Comment:
use `s.logger`, and make it an warning not a fatal level.

Reply:
done